compiler/rustc_expand/src/expand.rs

```diff
@@ -753,11 +753,8 @@ impl<'a, 'b> MacroExpander<'a, 'b> {
                         }
                     }
                 }
-                SyntaxExtensionKind::NonMacroAttr { mark_used } => {
+                SyntaxExtensionKind::NonMacroAttr { mark_used: _ } => {
```

Comment:
This was the only use of the `mark_used` field, `SyntaxExtensionKind::NonMacroAttr` can be made a unit variant now.